packages/v4/patternfly-docs/content/get-started/release-highlights.md

```diff
@@ -0,0 +1,111 @@
+---
+id: Release highlights
+title: Release highlights
+section: get-started
+---
+
+import './get-started.css';
+
+# 2023.06
+[View full details for this release on GitHub](https://github.com/patternfly/patternfly-react/releases).
+
+## Highlights 
+
+To upgrade your codebase to align with this release, [refer to our upgrade guide](/get-started/upgrade).
+
+### Dark theme support
+We are excited to officially support a dark theme on PatternFly. Not only  can you enable dark theme in your products; the PatternFly site can also be now be toggled from light to dark. 
+
+We've updated all of our color variables to ensure that your product is beautiful
+
+There are a couple of ways to enable dark theme in your products. You can: 
```

Comment:
On Friday we talked about the need to add further information into Developer Resources. @mcoker was looking at that. If so, we should link to it here. Also missing a "." at the end of this sentence.